blues/tests/test_mc.py

```diff
@@ -0,0 +1,128 @@
+
+import unittest, parmed
+import numpy as np
+from openmmtools.testsystems import get_data_filename
+from blues.moves import RandomLigandRotationMove
+from blues.engine import MoveEngine
+from blues.simulation import Simulation, SimulationFactory
+from simtk import openmm
+from openmmtools import testsystems
+import types
+import math
+
+class BLUESTester(unittest.TestCase):
+    """
+    Test the Simulation class.
+    """
+    def setUp(self):
```

Comment:
Combining the tests to use the same setup would be helpful. But in this case I'm using a different system in the tests (alanine in vacuum vs explicit solvent), and actually running some simulation steps for the vacuum system, so I'm leaning towards keeping them separate.